specification/hybridcompute/resource-manager/readme.md

```diff
@@ -410,4 +410,7 @@ suppressions:
     from: HybridCompute.json
     where: $.definitions.AgentVersion
     reason: Previously approved and released, would require potentially breaking changes
+  - code: GetCollectionResponseSchema
+    from: HybridCompute.json
+    reason: False positive
```

Comment:
please update the where clause to point to a specific instance instead of suppressing for the entire file